app-media-image-capture.html

```diff
@@ -454,6 +459,7 @@
             // user to configure it.
             // @see https://w3c.github.io/mediacapture-image/#redeyereduction-section
             if (this.redEyeReduction != null &&
+                this.photoCapabilities &&
```

Comment:
is this needed? we should've bailed out on L450 if it was null/undefined, no? Or is undefined ok?

Reply:
Will fix.